src/draw_buffer.rs

```diff
@@ -6,10 +6,25 @@ use std::rc::Rc;
 use GLContext;
 use NativeGLContextMethods;
 
-#[derive(Debug)]
+#[cfg(target_os="macos")]
+use io_surface::IOSurfaceID;
+#[cfg(target_os="macos")]
+use core_foundation::base::TCFType;
+#[cfg(target_os="macos")]
+use core_foundation::dictionary::CFDictionary;
+#[cfg(target_os="macos")]
+use core_foundation::string::CFString;
+#[cfg(target_os="macos")]
+use core_foundation::number::CFNumber;
+#[cfg(target_os="macos")]
+use core_foundation::boolean::CFBoolean;
```

Comment:
I would move most of these into the match arm that uses them, which will allow us to remove the #[cfg] attributes.